commands/dbq.bee.inc

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
```

Comment:
I think it would make sense to add this to `db.bee.inc`

Reply:
OK